deps/spidershim/spidermonkey/mozglue/build/moz.build

```diff
@@ -6,10 +6,7 @@
 
 # Build mozglue as a shared lib on Windows, OSX and Android.
 # If this is ever changed, update MOZ_SHARED_MOZGLUE in browser/installer/Makefile.in
-if CONFIG['OS_TARGET'] in ('WINNT', 'Darwin', 'Android'):
-    SharedLibrary('mozglue')
-else:
-    Library('mozglue')
+SharedLibrary('mozglue')
```

Comment:
I see, makes sense.  But the right fix is to link against mozglue statically on Linux.  Fixed that in 2df4ce2225.